pkg/worker/types.go

```diff
@@ -11,11 +11,12 @@ import (
 
 // EventProcessor provides a mechanism to act on events in the internal queue.
 type EventProcessor interface {
-	Process(events.Event) error
-	ShouldProcess(events.Event) (bool, *string)
+	MutateAppGateway(events.Event) error
+	MutateAKS(events.Event) error
+	ShouldMutateAppGateway(events.Event) (bool, *string)
 }
 
-// Worker listens on the eventChannel and runs the EventProcessor.Process
+// Worker listens on the eventChannel and runs the EventProcessor.MutateAppGateway
```

Comment:
This comments should be `Worker listens on the eventChannel and runs the EventProcessor.MutateAppGateway and EventProcessor.MutateAKS